src/Metadata/MetadataBase.php

```diff
@@ -249,7 +239,7 @@ public function setIsTrusted(bool $isTrusted): void
      */
     protected function ensureIsTrusted(bool $allowUntrustedAccess = false): void
     {
-        if (!$allowUntrustedAccess && !$this->isTrusted()) {
+        if (!$allowUntrustedAccess && !$this->isTrusted) {
```

Comment:
Why did this change?

Reply:
Since removing the setter I thought it would be good chance remove the getter method also since we don't use it outside of the class.